src/System.Net.Primitives/tests/FunctionalTests/System.Net.Primitives.Functional.Tests.csproj

```diff
@@ -42,6 +43,10 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
+    <!-- TODO: Remove this reference once packages are updated -->
```

Comment:
> I thought that packages are used from the local build

Nope

> If this intermediate hop is needed, shouldn't this reference the src/csproj instead?

I don't understand the question.  This is telling the test to use the local pkgproj, so it'll end up compiling against the ref built from that and running with the src built from that.